code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -115,15 +115,22 @@
 
 	. = ..()
 
+/obj/item/organ/internal/brain/mmi_holder/posibrain
+	name = "positronic brain"
+
 /obj/item/organ/internal/brain/mmi_holder/posibrain/New()
```

Comment:
The only reason why posibrain names were "correct" before is because it did a spawn, then set the name and such; that said, this spawn created issues, elsewhere, so it was removed---little did I know, that with how DNA is set, it ended up giving posibrains random human names.